File: week-02/day-02/06_printer.py
# - Create a function called `printer`
#   which prints the input parameters
#   (can have multiple number of arguments)

def printer_texts(*args):
    sumtext = ""
    for i in args:
        sumtext += i + " "
    print(sumtext)

def printer_nums(*args):
    sumnums = 0
    for i in args:
        sumnums += i
    print(sumnums)

printer_texts("Hello")
printer_texts("Hello", "GreenFox")
printer_texts("Hello", "GreenFox", "Good", "Day")
printer_nums(1 ,2 ,3 ,4, 5, 6, 7, 8 , 9, 10)

File: week-07/day-02/06_printer.js
'use strict';
// - Create a function called `printer`
//   which logs to the console the input parameters
//   (can have multiple number of arguments)

function printer() {
    let result = [];
    for (let i = 0; i < arguments.length; i++) {
        result.push(arguments[i]);
    }
    return result;
}

console.log(printer(1, 2, 3, 4 ,5 , 6, 7, 8, 9, 10));